modules/lvms-scaling-storage-of-clusters-using-rhacm.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-scaling-storage-of-clusters-using-rhacm_{context}"]
+= Scaling up the storage of clusters by using {rh-rhacm}
+
+You can scale up the storage capacity of worker nodes on the clusters by using {rh-rhacm}.
+
+.Prerequisites
+
+* You have access to the {rh-rhacm} cluster using an account with `cluster-admin` privileges.
+* You have created an `LVMCluster` custom resource (CR) by using {rh-rhacm}.
+* You have additional unused devices on each cluster to be used by {lvms-first}.
+
+.Procedure
+
+. Log in to the {rh-rhacm} CLI using your {product-title} credentials.
+. Edit the `LVMCluster` CR that you created using {rh-rhacm} by running the following command:
++
+[source,terminal]
+----
+$ oc edit -f <file_name> -ns <namespace> <1>
+----
+<1> Replace `<file_name>` with the name of the `LVMCluster` CR.
+
+. In the `LVMCluster` CR, add the path to the new device in the `deviceSelector` field.
++
+.Example `LVMCluster` CR
```

Comment:
```suggestion
.Example `LVMCluster` CR:
```

Reply:
Add a colon here to introduce the example CR

Reply:
Updated as per the suggestion.